packages/form/src/Field/Field.tsx

```diff
@@ -0,0 +1,197 @@
+import { Slot } from "@radix-ui/react-slot";
+import { TextField } from "@telegraph/text-field";
+import { Text as TelegraphText } from "@telegraph/typography";
+import { clsx } from "clsx";
+import React from "react";
+
+import { COLOR } from "./Field.constants";
+
+type RootProps = Omit<
```

Comment:
I'm just thinking out loud, but i'm guessing this component is meant to replace the internal implementation of `components/FormField/FormField.tsx`?

Reply:
Not understanding